AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/bookings-list/bookings-list.component.ts

```diff
@@ -479,5 +492,9 @@ export class BookingsListComponent implements OnInit, OnDestroy {
     ngOnDestroy() {
         this.$subcription?.unsubscribe();
         this.$ldSubcription?.unsubscribe();
+
+        if (this.$refreshSubscription) {
```

Comment:
might be nitpicking but you should be able to do `this.$refreshSubscription?.unsubscribe();` similar to the lines above